### ISSUE TITLE
[BUG] wild card in url resolves an error when upgrading to version 4

### ISSUE THREAD

**Issue by vlio20** [nestjs/swagger#416]
Hi,
I have a controller for static assets:
```
@Controller()
export class StaticAssetsController {

  @Get('site*')
  root(@Req() request: Request, @Res() response: Response) {
    response.sendFile(path.resolve('../webapp/dist/ui/index.html'));
  }
}
```

When upgrading to version 4, with the following configuration of the nest-cli:
```
{
  "collection": "@nestjs/schematics",
  "sourceRoot": "src",
  "compilerOptions": {
    "plugins": [
      {
        "name": "@nestjs/swagger/plugin",
        "options": {
          "classValidatorShim": true,
          "dtoFileNameSuffix": [".dto.ts"]
        }
      }
    ]
  }
}
```

I am getting the following error:
`(node:49514) UnhandledPromiseRejectionWarning: TypeError: Unexpected MODIFIER at 4, expected END`

By removing the wild card from the url the issue is resolved.

**Comment by thisissasa** [comment]
It was the same for me. It's just a temp quick fix but if you want to keep the wildcard you can put `@ApiExcludeEndpoint()` before the request method.

**Comment by kamilmysliwiec** [comment]
Fixed in 4.1.1. Thanks for reporting!
